python/mxnet/optimizer/optimizer.py

```diff
@@ -1244,6 +1244,54 @@ def update(self, index, weight, grad, state):
             kwargs = {}
             sgd_update(weight, grad, out=weight, lr=lr, wd=wd, **kwargs)
 
+
+@register
+class LAMB(Optimizer):
+    """LAMB Optimizer.
+    """
+    def __init__(self, learning_rate=0.001, beta1=0.9, beta2=0.999, epsilon=1e-6,
+                 lower_bound=None, upper_bound=None, bias_correction=False, **kwargs):
```

Comment:
can we set the default value of bias_correction to True?

Reply:
Earlier we discussed offline to keep it False. Any reason for changing it to True now ?